test/gtest/gtest_env_check_common.hpp

```diff
@@ -25,18 +25,11 @@
  *******************************************************************************/
 #pragma once
 
-#include <miopen/env.hpp>
 #include <gtest/gtest_common.hpp>
 
-MIOPEN_DECLARE_ENV_VAR_BOOL(MIOPEN_TEST_ALL)
-
 // For determining if we should run test suite. First ensure that test is supported on the hardware.
-// If the MIOPEN_TEST_ALL environment isn't set, then assume we are running standalone outside
-// CICD, and include the test. Otherwise, check the provided functor to ensure the environment
-// conditions match expected conditions to run this test suite.
-template <typename disabled_mask, typename enabled_mask, typename check_functor>
-bool ShouldRunTestCase(check_functor&& checkConditions)
+template <typename disabled_mask, typename enabled_mask>
+bool ShouldRunTestCase()
 {
-    return IsTestSupportedForDevMask<disabled_mask, enabled_mask>() &&
-           (!MIOPEN_TEST_ALL || checkConditions());
+    return IsTestSupportedForDevMask<disabled_mask, enabled_mask>();
 }
```

Comment:
I think now that we finished the clean-up this file can be removed, and all the ShouldRunTestCase() checks can be swapped to IsTestSupportedForDevMask<disabled_mask, enabled_mask>() directly.

I don't think we need to do it right now, but could simplify a bit.

Reply:
Done